server/model/board.go

```diff
@@ -111,6 +164,79 @@ type BoardMember struct {
 	SchemeViewer bool `json:"schemeViewer"`
 }
 
+func BoardFromJSON(data io.Reader) *Board {
+	var board *Board
+	_ = json.NewDecoder(data).Decode(board)
```

Comment:
I think we're trying to get away from ignoring JSON decode errors.

Reply:
Agree, but as this is following how the code currently works, I'd leave it for a refactor later as we'd need to account for errors in the client, which is the main user of these methods